doc/oauth-guide.md

```diff
@@ -33,6 +33,32 @@ Here we have added the `-v` flag to run APIcast in verbose mode, in order to all
 
 Our Authorization Server should also be up and running and ready to receive requests.
 
+## Running APIcast with OAuth (on OpenShift)
+
+APIcast in OAuth mode as mentioned previously has a Redis dependency that needs to be configured. In this case it will be necessary to deploy the Redis database separately using the template provided [here](https://github.com/3scale/apicast/tree/master/openshift).
+
+It's also worth mentioning that if you are using the provided Ruby client and Auth server then they must be exposed on a publicly available **DNS** to correctly route the requests from the OpenShift cluster. A free service like [Ngrok](https://ngrok.com/) is a useful tool for testing.
+
+## Pre-requisites
+
+- Cluster is up and running
+- New project has been created
+
+Proceed to the following steps if you have fulfilled the 2 previous requirements and then continue the set up as described in our [OpenShift tutorial](https://support.3scale.net/docs/deployment-options/apicast-openshift)
+
+### Steps
+
+The following commands are based on the **oc cluster up** installation.
+
+1. Log in as admin and create a persistent volume using `oc login -u system:admin` and `redis-persistent.json`
+  - mkdir -p /var/lib/docker/pv/01
+  - chmod g+w /var/lib/docker/pv/01
+  - chcon -Rt svirt_sandbox_file_t /var/lib/docker/pv/
+  - oc new-app --param PV=01 -f redis-persistent.json
```

Comment:
No. This is template that uses Persistent Volumes by Persistent Volume Claims. No Persistent Volumes are created and have to be created before creating Persistent Volume Claims.